tests/js-unit/formatting_test.ts

```diff
@@ -25,7 +27,7 @@ describe('formatting', () => {
 
                 expect(formatter(0)).to.equal('$0.00');
                 expect(formatter(1)).to.equal('$1.00');
-                expect(formatter(-1)).to.equal('-$1.00');
+                expect(formatter(-1)).to.equal('−$1.00');
```

Comment:
`d3-format` v2 changed the default minus sign
https://github.com/d3/d3-format/releases/tag/v2.0.0

Since this is presentational/formatting only wouldn't call this breaking.

Reply:
I'm glad `d3-format` has caught up, we've had to patch this for years in plotly.js to get the correct minus sign 😎